tensorflow/python/distribute/v1/BUILD

```diff
@@ -12,7 +12,7 @@ cuda_py_strict_test(
     name = "cross_device_ops_test",
     srcs = ["cross_device_ops_test.py"],
     tags = [
-        # "multi_and_single_gpu", # TODO(b/287692888): re-enable once the 2gpu test passes.
+        "multi_and_single_gpu", # TODO(b/287692888): re-enable once the 2gpu test passes.
```

Comment:
just notice this, ai think we can remove this comment since this test is passed?